src/cart/test/util/cart_logtest.py

```diff
@@ -297,7 +297,12 @@ def _check_pid_from_log_file(self, pid, abort_on_warning,
                         if line.rpc_opcode == '0xfe000000':
                             show = False
                     if show:
-                        show_line(line, 'NORMAL', 'warning in strict mode')
+                        # Allow WARNING or ERROR messages, but anything higher
+                        # like assert should trigger a failure.
+                        if line.level < cart_logparse.LOG_LEVELS['ERR']:
```

Comment:
Just to be clear, does this only run on the node local test or does it run on all of the unit tests.  We have some negative tests that purposely trigger asserts utilizing the capability of cmocka to test that a case fails

Reply:
Good point.  cart_logtest is supposed to be generic, and node_local_test is where the specific code is going, so this check would be better in node_local_test.  That said cart_logtest isn't used elsewhere yet so it's safe here, but special handling would be needed for that case when we implement this.